XML/TestCases/FunctionalTests-SGX.xml

```diff
@@ -12,9 +12,9 @@
     <SetupConfig>
       <TestLocation>uksouth</TestLocation>
       <SetupType>OneVM</SetupType>
-      <OverrideVMSize>Standard_DC2s_v2</OverrideVMSize>
+      <OverrideVMSize>Standard_DC2s_v2,=~Standard_DC[0-9]+s_v2</OverrideVMSize>
       <VMGeneration>2</VMGeneration>
-      <ARMImageName>Canonical UbuntuServer 18_04-lts-gen2 latest</ARMImageName>
+      <ARMImageName>=~UbuntuServer.*gen2</ARMImageName>
```

Comment:
If you introduce the new name convention of OverrideVMSize and ARMImageName, please add the new context in README or howtodo document.

Reply:
This is not newly introduced in this PR, but we should include more detailed instruction in ReadMe doc.  I have append changes in README, please take a look.